src/main/java/io/jenkins/plugins/opentelemetry/servlet/OpenTelemetryServletFilter.java

```diff
@@ -227,6 +235,36 @@ public void _doFilter(HttpServletRequest servletRequest, HttpServletResponse ser
             spanBuilder.setAttribute(SemanticAttributes.ENDUSER_ID, user.getId());
         }
 
+
```

Comment:
we may want to move this logic to a dedicated servlet filter because this servlet filter, in charge of creating traces of http requests can be disabled using the `otel.instrumentation.jenkins.web.enabled` config param.

See io.jenkins.plugins.opentelemetry.init.ServletFilterInitializer

Reply:
Sure, let me refine the code.